src/prefect/client/client.py

```diff
@@ -330,6 +330,18 @@ def _send_request(
             )
 
         # Check if request returned a successful status
+        if response.status_code == 400:
+            try:
+                msg = (
+                    f"400 Client Error: Bad Request for url: {url}\n\n"
+                    f"This is likely caused by a poorly formatted GraphQL query or mutation. "
+                    f"GraphQL sent:\n\n{parse_graphql(params)}"
+                )
+                raise ClientError(msg)
+            except ClientError:
+                raise
+            except Exception:
+                pass
```

Comment:
Would this 400 be handled by the caller then? Seems dangerous to silence exceptions in the message creation bit.

Reply:
In what sense is that dangerous?  The message is intended to add more info than already is presented, but if for some reason that's not possible it falls back to the original behavior